### ISSUE TITLE
My Token still shows as unknown in my Phantom Wallet

### ISSUE THREAD

**Issue by hybrisCole** [solana-labs/token-list#18309]
- **I'm submitting a ...**
  [ ] bug report
  [ ] feature request
  [ X] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
My token is still shown as Unknown on Phantom but it went thru the merge, I see logos and everything here https://solscan.io/token/6c1QMK2LCCojQ59uaormMLvDj3K8Vpfk7Br4fMAqgpqE#txs it looks like I added an extra comma in extensions sections but it was autofixed on the merge

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
- https://github.com/solana-labs/token-list/pull/18303

**Comment by limjame28** [comment]
Yeah i have the same problems my token still unknown in wallet but named in the sol scan

**Comment by hybrisCole** [comment]
hi, now I'm able to see my token info in Phantom, thank you :)

**Comment by limjame28** [comment]
how many hours did u wait?

**Comment by limjame28** [comment]
for me its been more than 24 hours only 400 supply but still unknown in my wallet :(

**Comment by hybrisCole** [comment]
Hey, it was 15hrs from now, I'm thinking it happened bc my PR build failed due to an extra comma

**Comment by hybrisCole** [comment]
I was looking at the PR you did and there's an error saying there's a dupe token ID 

https://github.com/solana-labs/token-list/runs/5157519777?check_suite_focus=true
https://github.com/solana-labs/token-list/runs/5158955133
https://github.com/solana-labs/token-list/runs/5157898143?check_suite_focus=true

maybe that's the reason of your error?

Regards,

**Comment by limjame28** [comment]
hmm the auto merge already done i can see it changing in solscan but not in my wallet hmm
